### PR TITLE
Fix eslint-plugin-promise docs location

### PR DESCRIPTION
**Pull request by macklinu**
https://github.com/xjamundx/eslint-plugin-promise/pull/112 moves the eslint-plugin-promise documentation from the README to a `docs/rules` directory. These paths are properly annotated in the `meta.docs.url` property for each rule, but until the next version of eslint-plugin-promise is released, we'll need to direct users the new URL in this PR.

### CODE REVIEW

**Comment by Arcanemagus** [comment]
Marking as blocked until that is actually merged and the links are valid.

**Comment by macklinu** [comment]
@Arcanemagus just merged https://github.com/xjamundx/eslint-plugin-promise/pull/112 - thank you!

File: plugins.json
Comment (Arcanemagus) [inline plugins.json]: Actually, this can now use the shorthand syntax of just `xjamundx` here.

**Review by Arcanemagus — CHANGES_REQUESTED** [review]

File: plugins.json
Comment (macklinu) [reply plugins.json]: Cool, will push up this change. 👍

**Review by Arcanemagus — APPROVED** [review]

**Pull request merged**
